django/db/backends/__init__.py

```diff
@@ -823,6 +823,15 @@ def cache_key_culling_sql(self):
         """
         return "SELECT cache_key FROM %s ORDER BY cache_key LIMIT 1 OFFSET %%s"
 
+    def case_cast_sql(self, db_type, internal_type):
```

Comment:
(To make the implication explicit: if not, perhaps this method shouldn't have `case` in its name.)